bin/deploy.sh

```diff
@@ -93,7 +100,8 @@ ln -s --force /data/www/task-runner/scripts/minute/* /etc/cron.minute/
 echo -e "* * * * *\troot\tcd / && run-parts --report /etc/cron.minute" >> /etc/crontab
 
 echo "Run database migrations"
-runuser -l vagrant -c "cd /data/www/library && php migrate.php"
+runuser -l vagrant -c "cd /data/www/library && php migrate-mysql.php"
+runuser -l vagrant -c "cd /data/www/library && php migrate-postgresql.php"
```

Comment:
Note that this assumes that this migration script will create the *schema* but not the *database*. [byjg/migration](https://packagist.org/packages/byjg/migration) does have that capability, but we're currently not using it; instead, the existing `library/base.sql` file creates the database, and earlier in this script we call `sudo mysql < base.sql`.